### DISCUSSION TITLE
Server returned HTTP Error 302

### DISCUSSION THREAD

**Discussion by nmathey** [FreshRSS/FreshRSS#6907] [Q&A]
Hello,

I'm trying to add a new feed from https://www.hbrfrance.fr/ with Xpath (Web scraping) with `//article` for "finding new items" and `descendant::div[@class="card-title"]` for item title.
However I got error stating that website might not accessible. When I tried to validate it, I got:
`Server returned HTTP Error 302: The HTTP server returned a redirect error that would lead to an infinite loop. The last 30x error message was: Moved Temporarily`

Any idea to move forward?

Thanks,

**Comment by Alkarex** [comment]
The Web site does some user-agent filtering. Try to import the following OPML:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<opml xmlns:frss="https://freshrss.org/opml" version="2.0">
	<head>
		<title>FreshRSS</title>
		<dateCreated>Wed, 16 Oct 2024 17:20:17 +0200</dateCreated>
	</head>
	<body>
		<outline text="Harvard Business Review France"
			type="HTML+XPath"
			xmlUrl="https://www.hbrfrance.fr/"
			htmlUrl="https://www.hbrfrance.fr/"
			description="RSS feed of Harvard Business Review France"
			frss:xPathItem="//article"
			frss:xPathItemTitle="descendant::div[@class='card-title']"
			frss:xPathItemContent="."
			frss:xPathItemUri="descendant::a/@href"
			frss:xPathItemAuthor="descendant::span[@class='card-author']"
			frss:xPathItemCategories="descendant::span[@class='card-category']"
			frss:xPathItemUid="descendant::a/@href"
			frss:cssFullContentFilter="button"
			frss:CURLOPT_USERAGENT="curl"
		/>
	</body>
</opml>
```

**Comment by nmathey** [comment]
Same issue: does it work on your side?

**Reply by Alkarex** [reply]
Yes, it worked for me.
However, I can see that their responses are quite erratic, and depend at least on user-agent, maybe other parameters.

https://validator.w3.org/nu/?doc=https%3A%2F%2Fwww.hbrfrance.fr%2F

![image](https://github.com/user-attachments/assets/d52e2ea2-f121-4368-8de9-e360504e057a)

Try to manually cURL from your server to find what might work

**Reply by nmathey** [reply]
Just tried several time directly from the server: it works without changing user-agent so far :(

**Reply by Alkarex** [reply]
> Just tried several time directly from the server: it works without changing user-agent so far :(

Do you mean with `curl` ?

**Reply by nmathey** [reply]
Yes

**Reply by Alkarex** [reply]
The bad redirection seems to be due to having to accept cookies. Try the cookies-related options

**Reply by nmathey** [reply]
Weird: I did enable cookies option and it didn't work the firsts few times. However, couple hours later it worked. Let's see in the long run.
